jsapp/js/account/security/accessLogs/accessLogsSection.component.tsx

```diff
@@ -4,15 +4,18 @@ import React from 'react';
 // Partial components
 // import Button from 'js/components/common/button';
 import PaginatedQueryUniversalTable from 'js/universalTable/paginatedQueryUniversalTable.component';
+import ExportToEmailButton from 'jsapp/js/components/exportToEmailButton/exportToEmailButton.component';
 
 // Utilities
-import useAccessLogsQuery, {type AccessLog} from './accessLogs.query';
+import useAccessLogsQuery, {useExportAccessLogs, type AccessLog} from './accessLogs.query';
 import {formatTime} from 'js/utils';
 // import sessionStore from 'js/stores/session';
 
 // Styles
 import securityStyles from 'js/account/security/securityRoute.module.scss';
 
+const exportData = useExportAccessLogs();
```

Comment:
I believe it could be better to not use a hook here, since we're not getting any benefit from it being a hook, and just export the `startAccessLogsExport`. Then we don't even need it here, we could just use it directly in the button's `onclick`